main.go

```diff
@@ -142,6 +156,33 @@ func messageCreate(s *discordgo.Session, m *discordgo.MessageCreate) {
 			outputJSON := util.FormatEarnings(earnings)
 
 			s.ChannelMessageSend(m.ChannelID, outputJSON)
+		} else if action, _ := regexp.MatchString("(?i)^!remindme$", slice[0]); action {
+			messageArr := strings.Split(m.Content, "\"")
+			if len(messageArr) != 3 {
+				s.ChannelMessageSend(m.ChannelID, "Reminder messages must be surrounded by quotes \"{message}\" ")
+				return
+			}
+			// We store the person who sent the message as well as the channel id into the redis cache so we know where and who to contact later
+			message := m.ChannelID + "~*" + m.Author.Mention() + ": " + messageArr[1]
+			date := slice[len(slice) - 1]
+			match, _ := regexp.MatchString("(0?[1-9]|1[012])/(0?[1-9]|[12][0-9]|3[01])/(\\d\\d)", date)
+			if match == false {
+				s.ChannelMessageSend(m.ChannelID, "Invalid date given loser")
+				return
+			}
+
+			dateCheck, _ := ime.Parse("01/02/06", date)
```

Comment:
is the date 01/02/06 meaningful?

Reply:
yuh
https://pauladamsmith.com/blog/2011/05/go_time.html